Resources/public/js/views/tabs/ez-locationviewdetailstabview.js

```diff
@@ -0,0 +1,183 @@
+                this.render();
+            });
+
+            this.after('ownerChange', function (e) {
+                this.render();
+            });
+        },
+
+        render: function () {
+            var container = this.get('container'),
+                content = this.get('content'),
+                currentVersion = content.get('currentVersion'),
+                translationList = currentVersion.getTranslationsList();
+
+            container.setHTML(this.template({
+                "content": content.toJSON(),
+                "location": this.get('location').toJSON(),
+                "currentVersion": currentVersion.toJSON(),
+                "lastContributor": this.get('creator').toJSON(),
+                "contentCreator": this.get('owner').toJSON(),
+                "translationList": translationList,
+                "languageCount": translationList.length,
+            }));
+
+            return this;
+        },
+
+        /**
+         * Fire the `loadUser` event
+         * @method _fireLoadUser
+         * @protected
+         */
+        _fireLoadUser: function () {
+            var creatorId = this.get('content').get('currentVersion').get('resources').Creator,
+                ownerId = this.get('content').get('resources').Owner;
+
+            /**
+             * Fired when the details view needs authors
+             * @event loadUser
+             * @param {String} userId Id of the author
+             * @param {String} attributeName Where to store the result
+             */
+            this.fire('loadUser', {
+                userId: creatorId,
+                attributeName: 'creator',
+            });
+
+            if (creatorId == ownerId) {
+                this.onceAfter('creatorChange', function (e) {
+                    this.set('owner', this.get('creator'));
+                });
+            } else {
+                this.fire('loadUser', {
+                    userId: ownerId,
+                    attributeName: 'owner',
+                });
+            }
+        },
+
+    }, {
+        ATTRS: {
+            /**
+             * The title of the tab
+             *
+             * @attribute title
+             * @type {String}
+             * @default "Details"
+             * @readOnly
+             */
+            title: {
+                value: "Details",
+                readOnly: true,
+            },
+
+            /**
+             * The identifier of the tab
+             *
+             * @attribute identifier
+             * @type {String}
+             * @default "details"
+             * @readOnly
+             */
+            identifier: {
+                value: "details",
+                readOnly: true,
+            },
+
+            /**
+             * The creator of the content
+             *
+             * @attribute creator
+             * @type {Object}
+             */
+            creator: {
+                valueFn: function () {
```

Comment:
you should not create an instance of Y.eZ.User as the default value of those attributes as they will be overwritten by the plugin. (That means you have to handle a non object value in `render`)